pkgs/tools/inputmethods/keyd/default.nix

```
@@ -0,0 +1,32 @@
+{ lib, stdenv, python39Packages, python39, fetchFromGitHub }:
```

Either this or change the buildInputs below.

```suggestion
{ lib, stdenv, python3Packages, python3, fetchFromGitHub }:
```